.prettierrc.yml

```
@@ -0,0 +1,7 @@
+bracketSpacing: true
+printWidth: 120
+proseWrap: "always"
+singleQuote: false
+tabWidth: 2
+trailingComma: "all"
+useTabs: false
```

```suggestion
printWidth: 120
tabWidth: 4
useTabs: false
semi: true
singleQuote: true
trailingComma: none
arrowParens: always
bracketSpacing: false

overrides:
  - files: '*.sol'
    options:
      printWidth: 100
      tabWidth: 4
      useTabs: false
      singleQuote: false
  - files: '*.md'
    options:
      printWidth: 100
      tabWidth: 2
      useTabs: false
      singleQuote: true

```

See https://github.com/aragon/osx/blob/develop/.prettierrc

---

Now we run into problems with `yarn lint:sol`...